### DISCUSSION TITLE
Need examples for creating custom features for multivariate time series

### DISCUSSION THREAD

**Discussion by bharathklogan** [blue-yonder/tsfresh#989] [Ideas]
Hi Team,

Can you please make an example in the tsfresh documentation on how to create custom features for the multivariate time series?
And, also for creating custom features for multiple time series?

This will be really useful for people like me who are working in different variants of time series data.

Thanks,
Bharath.

**Comment by nils-braun** [comment]
Hi Bharath! Thanks for your request. tsfresh does not support multivariate features (yet). Therefore we also have no example on its usage. All feature calculators are only using information on a single time series.
